viz/PointcloudVisualization.cpp

```diff
@@ -32,6 +33,11 @@ osg::ref_ptr< osg::Node > PointcloudVisualization::createMainNode()
     drawArrays = new osg::DrawArrays( osg::PrimitiveSet::POINTS, 0, pointsOSG->size() );
     pointGeom->addPrimitiveSet(drawArrays.get());
 
+    //This fixes the Visualization for one point
+    //See http://forum.openscenegraph.org/viewtopic.php?t=6980&view=previous
+    osg::View* view = getWidget()->getView(0);
+    view->getCamera()->setCullingMode(view->getCamera()->getCullingMode() & ~osg::CullSettings::SMALL_FEATURE_CULLING);
```

Comment:
Wouldn't it be possible to do it on a per-subgraph basis (instead of globally). This change can have a pretty heavy (performance) effect on whole scenes.